### DISCUSSION TITLE
Maximum width bound

### DISCUSSION THREAD

**Discussion by bryanchenmchp** [eclipse-theia/theia#13098] [Q&A]
Suppose I want to make a widget that contains a split layou in the bottom panel, is it possible to add a maximum width constraint to a pane of the split layout?

**Comment by martin-fleck-at** [comment]
I don't think there is anything like that out of the box so you would probably need to customize the SplitLayout yourself, in particular the `moveHandle` function (see [here](https://github.com/phosphorjs/phosphor/blob/master/packages/widgets/src/splitlayout.ts#L237)) that takes care of the actual movement and resizing of the elements.
